monitor.c

```diff
@@ -419,9 +420,12 @@ monitor_read_log(struct monitor *pmonitor)
 	struct sshbuf *logmsg;
 	u_int len, level, forced;
 	char *msg;
+	char *pname;
 	u_char *p;
 	int r;
 
+	u_int sftp_log_level, sftp_log_facility, sftp_log_stderr;
```

Comment:
move this and line 423 close to where you use them. and encapsulate in windows block